cli/daemon/run/testdata/echo_client/client.ts

```diff
@@ -26,32 +26,14 @@ export default class Client {
     public readonly test: test.ServiceClient
 
 
-    /**
-     * @deprecated This constructor is deprecated, and you should move to using BaseURL with an Options object
-     */
-    constructor(target?: string, token?: string)
-
```

Comment:
Do we no longer support the old-style constructor?

Reply:
It's still added - but only when token based authentication is in use